nominees/our-world-in-data.json

```diff
@@ -3,47 +3,134 @@
+      "evidenceURL": "https://sdg-tracker.org/gender-equality"
+    },
+    {
+      "SDGNumber": 6,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/water-and-sanitation"
+    },
+    {
+      "SDGNumber": 7,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/energy"
+    },
+    {
+      "SDGNumber": 8,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/economic-growth"
+    },
+    {
+      "SDGNumber": 9,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/infrastructure-industrialization"
+    },
+    {
+      "SDGNumber": 10,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/inequality"
+    },
+    {
+      "SDGNumber": 11,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/cities"
+    },
+    {
+      "SDGNumber": 12,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/sustainable-consumption-production"
+    },
+    {
+      "SDGNumber": 13,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/climate-change"
+    },
+    {
+      "SDGNumber": 14,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/oceans"
+    },
+    {
+      "SDGNumber": 15,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/biodiversity"
+    },
+    {
+      "SDGNumber": 16,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/peace-justice"
+    },
     {
       "SDGNumber": 17,
-      "evidenceText": "Related to Target 17.6 - Enhances knowledge sharing, specifically making this data available to individuals globally.",
-      "evidenceURL": "https://en.wikipedia.org/wiki/Our_World_in_Data"
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/global-partnerships"
     }
   ],
   "sectors": [
-    "Digital/Data/Tech",
-    "Fair & Responsible Media",
-    "Media",
-    "Research"
+    "Digital/Data/Tech"
   ],
   "type": [
+    "content",
+    "software",
     "data"
   ],
   "repositories": [
     {
-      "name": "main",
-      "url": ""
+      "name": "owid-grapher",
+      "url": "https://github.com/owid/owid-grapher"
+    },
+    {
+      "name": "owid/covid-19-data",
+      "url": "https://github.com/owid/covid-19-data"
```

Comment:
The software component and scripts are unlicensed and should be assigned an open source license, if it is to be certified:

https://github.com/owid/covid-19-data/tree/master/scripts

Compare to the data component, which is more clearly licensed:

https://github.com/owid/covid-19-data/tree/master/public/data#license

Reply:
The [Grapher codebase](https://github.com/owid/owid-grapher) is already licensed under MIT (very open). But the OWID website can’t be open licensed as a whole, because they are using data from various providers that are included in the repos (can’t relicense third-party data files). Something similar happens with the Covid repo - they’re happy to distribute the scripts under open license but we can’t license the data from third parties.

Cc: @jwflory

Reply:
@nathanbaleeta There must have been a misunderstanding. I am definitely not asking them to re-license third-party data files; it isn't possible! 🙂 

If the `scripts/` directory is…

1. Distributed with a `LICENSE.txt` file, and
2. the README.md notes that the license applies to the scripts/software component,

…then I would approve this (together with my previous comment about `sdg-tracker.org`.

As it stands, this submission is asking us to certify something that does not meet indicator no. 2 of the DPG Standard. I don't think the onus should be on the DPG reviewer team or a DPG Registry visitor to guess what is open and what is not; it should be clearly noted by a license file distributed alongside the source with a narrative component or FAQ to distinguish how and what the open source license applies to, and what it does not.

Essentially, we are asking them to make an implicit dual-license situation into an explicit one.